src/components/dashboard/SendQRSummary.js

```diff
@@ -44,7 +44,23 @@ const SendQRSummary = ({ screenProps }: AmountProps, params) => {
     setProfile(profile)
   }
 
-  const confirm = async () => ((await goodWallet.isCitizen()) ? sendGD() : faceRecognition())
+  const confirm = async () => {
+    try {
+      if (await goodWallet.isCitizen()) {
+        sendGD()
+      } else {
+        faceRecognition()
+      }
+    } catch (e) {
+      log.error('Send TX failed:', e.message, e)
+      showErrorDialog({
+        visible: true,
+        title: 'Transaction Failed!',
+        message: `There was a problem sending G$. Try again`,
+        dismissText: 'OK',
```

Comment:
can the user retry here? or do we need to redirect him to the begining (ie gotoroot)?
check this for all dialogs where we ask to try again

Reply:
I set retry here and in SendLinkSummary.js